win32ss/user/ntuser/timer.c

```diff
@@ -17,7 +17,9 @@ static LIST_ENTRY TimersListHead;
 static LONG TimeLast = 0;
 
 /* Windows 2000 has room for 32768 window-less timers */
-#define NUM_WINDOW_LESS_TIMERS   32768
+/* This settings give ids of timers 256-32767, same as windows */
+#define NUM_WINDOW_MAX_TIMERS   32767 // 32768 - 1
+#define NUM_WINDOW_LESS_TIMERS   32512 // 32768 - 256
```

Comment:
```suggestion
/* These values give timer IDs [256,32767], same as on Windows */
#define NUM_WINDOW_MAX_TIMERS   (32768 - 1)
#define NUM_WINDOW_LESS_TIMERS  (32768 - 256)
```
That `NUM_WINDOW_MAX_TIMERS` may be named instead something like `MAX_WINDOW_TIMER_ID` if this value is really a particular ID value, not a number of timers.